spec/features/drivers/user_can_edit_driver_spec.rb

```diff
@@ -1,14 +1,19 @@
 require 'rails_helper'
 
-feature 'user can edit driver' do
+feature 'admin can edit driver' do
+  let!(:admin) { create(:admin, email: 'email@email.com', password: 'password') }
   let!(:driver) { create(:driver) }
 
   background do
+    visit(new_session_path)
+    fill_in('Email', with: 'email@email.com')
+    fill_in('Password', with: 'password')
+    click_button('Login')
```

Comment:
i see this repeated too much, we should refactor this so we dont have to write this every time.

we can either have a helper / or shared setup for the tests that does this.
or we set a session in the backend when a user is already admin.